feat: Add support for inverting reset and boot pins (ESF-177) (#120)
espressif/esp-serial-flasher: README.md

Default: 50

* `SERIAL_FLASHER_RESET_INVERT`

This inverts the output of the reset gpio pin. Useful if the hardware has inverting connection

> between the host and the target reset pin. Works only on UART interface.

Reviewer: Just a nitpick.
```suggestion
between the host and the target reset pin. Implemented only for UART interface.
```

Default: n

* `SERIAL_FLASHER_BOOT_INVERT`
This inverts the output of the boot (IO0) gpio pin. Useful if the hardware has inverting connection

> between the host and the target boot pin. Works only on UART interface.

Reviewer: ```suggestion
between the host and the target boot pin. Implemented only for UART interface..
```

Default: n

> 

Reviewer: Rewrite appropriately.
